crypto/keys/der/der_test.go

```diff
@@ -111,6 +118,8 @@ func TestNewProtoFromSpec(t *testing.T) {
 		},
 	} {
 		t.Run(test.desc, func(t *testing.T) {
+			t.Parallel()
+
 			pb, err := NewProtoFromSpec(test.keySpec)
```

Comment:
I think this will fall foul of https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables.